### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by MrCatBoss** [MrCatBoss/Neuro-tools#2]
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from keras.models import Sequential
from keras.layers import Dense
from keras.optimizers import Adam

# Загрузка данных из csv файлов
df = pd.read_csv('train.csv')

# Разделение данных на обучающую и тестовую выборки
train_data, test_data, train_target, test_target = train_test_split(df.iloc[:, :-1], df.iloc[:, -1], test_size=0.2, random_state=42)

# Создание нейронной сети
model = Sequential()
model.add(Dense(128, input_dim=train_data.shape[1], activation='relu'))
model.add(Dense(64, activation='relu'))
model.add(Dense(1, activation='linear'))

# Компиляция модели с определенными параметрами оптимизации и функции потерь
model.compile(loss='mean_squared_error', optimizer=Adam(lr=0.01))

# Обучение модели на обучающих данных
history = model.fit(train_data, train_target, epochs=100, batch_size=32, validation_split=0.2)

# Оценка ошибки на тестовых данных
score = model.evaluate(test_data, test_target, batch_size=32)

# Визуализация результатов обучения и тестирования модели
import matplotlib.pyplot as plt

plt.plot(history.history['loss'], label='train')
plt.plot(history.history['val_loss'], label='validation')
plt.legend()
plt.show()

import pandas as pd
import tensorflow as tf

# загрузка данных из csv файла
df = pd.read_csv('data.csv')

# подготовка данных для обучения
train_data = df.iloc[:, :-1].values
train_labels = df.iloc[:, -1].values

# создание модели нейронной сети
model = tf.keras.Sequential([
    tf.keras.layers.Dense(64, activation='relu', input_shape=(train_data.shape[1],)),
    tf.keras.layers.Dense(64, activation='relu'),
    tf.keras.layers.Dense(1)
])

# компиляция модели
model.compile(optimizer='adam', loss='mean_squared_error')

# обучение модели
model.fit(train_data, train_labels, epochs=100, validation_split=0.2)

# оценка модели на тестовых данных
test_data = [[...], [...], ...]  # тестовые данные
test_labels = [...]  # правильные ответы
test_loss = model.evaluate(test_data, test_labels)

# вывод результатов
print('Test loss:', test_loss)
